browser-test/src/admin_api_keys.test.ts

```diff
@@ -34,14 +34,24 @@ describe('Managing API keys', () => {
 
     const credentials = await adminApiKeys.createApiKey({
       name: 'Test API key',
-      expiration: '2022-01-31',
-      subnet: '8.8.8.8/32',
+      expiration: '2100-01-01',
```

Comment:
Not blocking: Was this failing before the change or was it passing due to an application logic bug?

Reply:
Phew. That's pretty much the best outcome :D